build/template-build-and-prep-automation.yaml

```diff
@@ -96,8 +96,73 @@ steps:
     Contents: '**/bin/**/*'
     TargetFolder: '$(build.artifactstagingdirectory)/DeviceAuth'
 
+- task: CopyFiles@2
+  displayName: 'Stage WAM DevApp Dlls'
+  inputs:
+    SourceFolder: tests/devapps/wam
+    Contents: '**/bin/**/**'
+    TargetFolder: '$(build.artifactstagingdirectory)/WAM'
+
 - task: PublishPipelineArtifact@1
   displayName: 'Publish Artifact: drop'
   inputs:
     targetPath: $(Build.artifactstagingdirectory)
     artifactName: drop
+
+- task: PowerShell@2
+  displayName: 'Verify Existence of Runtime Dlls'
+  inputs:
+    targetType: 'inline'
+    script: |
+        #Search Folder
+        $searchInFolder = "\WAM\";
+        $searchInFolder = Join-Path $(build.artifactstagingdirectory) $searchInFolder 
+
+        Write-Host $searchInFolder 
+
+        #runtime path with file name
+        $runtimes = @('runtimes\win-x64\native\msalruntime.dll', 
+                      'runtimes\win-x86\native\msalruntime_x86.dll', 
+                      'runtimes\win-arm64\native\msalruntime_arm64.dll');
+
+        #dev apps to check
+        $AppsToCheck = @('MSIX\WPF\bin\Release\', 
+                         'MSIX\WPF.Package\bin\AnyCPU\Release\WPF\', 
+                         'NetCoreWinFormsWam\bin\Release\net6.0-windows10.0.17763.0\', 
```

Comment:
Would wildcard work for these? Making sure this script is robust in case we change dev apps.
Something like `'NetCoreWinFormsWam\bin\Release\*''` or ``'*\bin\Release\*''``